src/index.css

```diff
@@ -1,12 +1,12 @@
 :root {
-  --main-color: #8aba87;
+  --main-color: #f4f4f4;
   --anti-color: #b788ba;
```

Comment:
please be more specific :) so for example are those for the background, primary button, secondary button?

Reply:
that concerns the color-topic. One Question: what do we write in index.css? It overwrites stuff later so it has to be generally accepted.

Reply:
you're right, there shouldn't be anything in `index.css`, I'd make a new css file only for the colors and import in `app.css`. If it also needs to be imported in the files where it's used then we need to do that :)